app/javascript/packs/outdoor_map_builder.js

```diff
@@ -172,6 +172,7 @@ window.routingControl = L.Routing.control({
 // when routing call happens, there will be the stop button in the navigation plan
 .on('routingstart', (e)=>{
     document.getElementById('StopNavigation').style.display = 'block';
+    document.getElementById('welcome-routing-text').style.display = 'none';
```

Comment:
In the beginning I was a little confused as to what this does, maybe you could update the name to refer to this being for the mobile view?